packages/turbo-benchmark/src/templates/jest.config.ts

```diff
@@ -1,4 +1,6 @@
-module.exports = {
+import type { Config } from "jest";
+
+const config = {
```

Comment:
Out of curiosity why `satisfies` instead of:

`const config: Config = { ...`

Reply:
because it won't narrow the type.  this came up as an error in `examples/with-nestjs/packages/jest-config/base.ts`.  try changing it there like `const config: Config = {` and you'll see an error in `next.ts`.

originally I just did `: Config` but after seeing that error I thought: "oh, why not, there's no harm in letting the type stay narrow rather than widening it"

Reply:
Got it, thanks!